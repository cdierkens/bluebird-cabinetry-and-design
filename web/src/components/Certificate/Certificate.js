import PropTypes from "prop-types";
import React from "react";
import { buildImageObj } from "../../../src/lib/helpers";
import { imageUrlFor } from "../../../src/lib/image-url";
import styles from "./Certificate.module.css";

const Certificate = ({ image }) => {
  return (
    <div className={styles.Certificate}>
      <img
        src={imageUrlFor(buildImageObj(image))
          .width(1200)
          .height(Math.floor((9 / 16) * 1200))
          .fit("crop")
          .auto("format")
          .url()}
        alt={image.alt}
      />
    </div>
  );
};

Certificate.propTypes = {
  image: PropTypes.object,
};

export default Certificate;
