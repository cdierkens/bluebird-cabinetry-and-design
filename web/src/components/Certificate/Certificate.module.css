.Certificate {
  /* Plopped styles for Certificate */
}
