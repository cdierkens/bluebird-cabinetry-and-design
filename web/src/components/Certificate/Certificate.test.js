import { render } from "@testing-library/react";
import React from "react";
import Certificate from "./Certificate";

it("matches snapshot", () => {
  const { asFragment } = render(<Certificate />);

  expect(asFragment()).toMatchSnapshot();
});
