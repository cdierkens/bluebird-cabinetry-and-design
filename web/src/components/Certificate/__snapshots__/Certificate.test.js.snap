// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Certificate"
  >
    <img
      src="https://cdn.sanity.io/images/mock-id/mock-dataset/mock-100x100.jpg?rect=0,21,100,56&w=1200&h=675&fit=crop&auto=format"
    />
  </div>
</DocumentFragment>
`;
